docs/Installation.md

```diff
@@ -0,0 +1,15 @@
+## Prerequisites
```

Comment:
I'm not sure I agree on this. Since running Quickstart also requires Installation. Specifically we created Quickstart so that users can try basic things out without going into details in Usage.